src/main/java/com/eleks/academy/whoami/core/impl/PersistentGame.java

```diff
@@ -169,7 +170,7 @@ public void answerQuestion(String player, QuestionAnswer questionAnswer) {
             answeringPlayer.setEnteredAnswer(true);
             answeringPlayer.setPlayerAnswer(String.valueOf(questionAnswer));
 
-            history.setAllAnswers(answeringPlayer.getNickname(), questionAnswer);
+            addAnswerToHistory(answeringPlayer.getNickname(), questionAnswer.toString());
```

Comment:
Explicit usage of toString() method is not a good practice since it can produce NullPointerException and it is not fully clear what will be the result of toString() (is it overridden or not, etc), better to pass it as a whole object or call specific getter method if needed.